src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/DataController.cs

```diff
@@ -326,7 +339,6 @@ private DataElement CreateDataElement(HttpRequest request, string elementType, I
 
             string filePath = DataFileName(instance.AppId, instanceGuid.ToString(), newData.Id.ToString());
```

Comment:
Why do you need ToString() on newData.Id?

Reply:
Removed, thanks.